tests/integration/test_documentation_files_validation.py

```
@@ -0,0 +1,357 @@
+        
+        # Check for consistent spelling/terminology
+        inconsistencies = {
+            'pytest': ['py.test', 'py-test'],
+            'frontend': ['front-end', 'Front-end'],
+            'backend': ['back-end', 'Back-end'],
+        }
+        
+        for md_file in md_files:
+            content = md_file.read_text(encoding='utf-8')
+            
+            for correct, alternatives in inconsistencies.items():
```

Ruff (0.14.8)

Loop control variable `correct` not used within loop body

Rename unused `correct` to `_correct`

(B007)